test/integration/rules/td-headers-attr/td-headers-attr.html

```diff
@@ -32,6 +37,21 @@
   <td id="self" headers="self">World</td>
 </table>
 
+<table id="fail4">
+  <td id="hdr1">Hello</td>
+  <td headers="hdr1">World</td>
+</table>
+
+<table id="fail5">
+  <th role="cell" id="th-role-cell-hdr">Hello</td>
```

Comment:
> So i updated .prettierignore, let me know if there is better way to fix it.

I'm pretty sure this was just a typo in his suggestion; this is intended to be a case exercising that the rule correctly fails if a `headers` attribute. points to a `th` element with a non-heading role, so I think it's correct for it to just be a `th` element. Shouldn't need to add it to `.prettierignore`, this is just prettier correctly catching a typo.

```suggestion
  <th role="cell" id="th-role-cell-hdr">Hello</th>
```

Reply:
Ok, fyi @WilcoFiers